src/Aspire.Hosting.AWS/Lambda/APIGatewayEmulatorResource.cs

```diff
@@ -0,0 +1,26 @@
+﻿// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+
+using Aspire.Hosting.ApplicationModel;
+using k8s.KubeConfigModels;
+
+namespace Aspire.Hosting.AWS.Lambda;
+
+
+/// <summary>
+/// Resource representing the Amazon API Gateway emulator.
+/// </summary>
+/// <param name="name">Aspire resource name</param>
+public class APIGatewayEmulatorResource(string name, APIGatewayType apiGatewayType) : ExecutableResource(name, 
+        "dotnet",
+        Environment.CurrentDirectory
+        )
+{
+    internal void AddCommandLineArguments(IList<object> arguments)
+    {
+        arguments.Add("lambda-test-tool");
```

Comment:
isn't the name of our tool `dotnet-lambda-test-tool`?

Reply:
apparently ` dotnet lambda-test-tool` works and so does `dotnet-lambda-test-tool`.

Reply:
Discussed in sync meeting and we are on the same page now.